make_queue/util/user_info.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth.models import User
+import ldap
+
+LDAP_HOST = 'ldap://at.ntnu.no'
+LDAP_BASE = 'dc=ntnu, dc=no'
+
+LDAP_FIELDS = {
+    'full_name': 'cn',
+    'surname': 'sn',
+    'group_id': 'gidNumber',
+    'id': 'uidNumber',
+    'username': 'uid',
+    'mail': 'mail',
+}
+
+
+def LDAP_search(field, value):
+    l = ldap.initialize(LDAP_HOST, bytes_mode=False)
+    l.simple_bind()
+    query = "({}={})".format(LDAP_FIELDS[field], value)
+    return l.search_s(LDAP_BASE, ldap.SCOPE_SUBTREE, query)
+
+
+def get_LDAP_field(ldapData, field):
+    return ldapData[0][1].get(LDAP_FIELDS[field], [b''])[0].decode()
+    # list<tuple<str, dict<str, list<bstr>>>>
```

Comment:
I think it would be more useful with a description of the data structure (how is everything packed) we receive from LDAP, rather than just the datatypes. E.g. "We get a list of results. These consists of tuples of...".